src/uu/cp/src/cp.rs

```diff
@@ -230,6 +224,15 @@ pub struct Options {
     verbose: bool,
 }
 
+// From /usr/include/linux/fs.h:
+// #define FICLONE		_IOW(0x94, 9, int)
+#[cfg(any(target_os = "linux", target_os = "android"))]
+macro_rules! FICLONE {
+    () => {
+        0x40049409
+    };
+}
```

Comment:
Why does this have to be a macro? Would this work?
```rust
const FICLONE: libc::c_ulong = 0x40049409;
```

Reply:
Because the libc expects different type depending on the CPU (32 vs 64)

Reply:
I added a comment

Reply:
But using `libc::c_ulong` would fix that right? That's defined as `u32` or `u64` depending on the arch.

Reply:
And with the conversation:
```
error: useless conversion to the same type: `u64`
    --> src/uu/cp/src/cp.rs:1572:17
     |
1572 |                 FICLONE.try_into().unwrap(),
     |                 ^^^^^^^^^^^^^^^^^^
     |
     = note: `-D clippy::useless-conversion` implied by `-D warnings`
     = help: consider removing `.try_into()`
     = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#useless_conversion
```